### PR TITLE
Wappalyzer Update 2021-08-03

### PR DESCRIPTION
**Pull request by zapbot**
Updates based on AliasIO/Wappalyzer@cfad643

Signed-off-by: zapbot <12745184+zapbot@users.noreply.github.com>

### CODE REVIEW

**Review by thc202 — APPROVED** [review]

File: addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json [hunk 1]
```diff
@@ -328,15 +340,25 @@
         1,
         6
       ],
+      "cookies": {
```
Comment (thc202) [inline addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json:343]: This does not seem to be handled.

Comment (kingthorin) [reply addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json:343]: It doesn’t seem supported by the upstream schema either (array or non-blank non-empty string).

Comment (kingthorin) [reply addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json:343]: Edit: I'm wrong it is an object upstream.

**Review by kingthorin — APPROVED** [review]

File: addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json (hunk 1 above)
Comment (kingthorin) [reply addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json:343]: I'll tackle it. If I don't get it done shortly I'll open a new issue.

**Pull request merged**